integrations/java/install-driver.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -o errexit
+
+export JAVA_HOME="/opt/java/jdk11"
+
+cd mongo-java-driver || exit
+
+./gradlew --info driver-workload-executor:compileJava
```

Comment:
This will end up downloading a lot of stuff from Maven Central as well as compiling the whole driver.  Typically takes about 2 minutes, so better to do it here than in the executor.

Reply:
Ack.